modules/ctransformers_model.py

```diff
@@ -49,6 +47,7 @@ def decode(self, ids):
 
     def generate(self, prompt, state, callback=None):
         prompt = prompt if type(prompt) is str else prompt.decode()
+        # ctransformers uses -1 for random seed
         generator = self.model._stream(
```

Comment:
Please use `self.model()` as `self.model._stream()` is an internal method and should not be used directly.